common/lib/xmodule/xmodule/course_module.py

```diff
@@ -437,152 +439,308 @@ class CourseFields(object):
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Deleting Course Content"),
+                        "short_description": _("Ensure video quality"),
                         "long_description": _(
-                            "Delete Sections, Subsections, or Units you don't need anymore. Be careful, as there is "
-                            "no Undo function."
+                            "Ensure that all course videos are of consistent quality. Also ensure that audio levels"
+                            "of videos throughout the course are consistent and do not vary dramatically."
                         ),
                         "is_checked": False,
                         "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Add an Instructor-Only Section to Your Outline"),
+                        "short_description": _("Provide transcripts"),
                         "long_description": _(
-                            "Some course authors find using a section for unsorted, in-progress work useful. To do "
-                            "this, create a section and set the release date to the distant future."
+                            "Ensure that transcripts are uploaded for all video and audio content."
                         ),
                         "is_checked": False,
                         "action_url": "CourseOutline",
-                        "action_text": _("Edit Course Outline"),
+                        "action_text": _("View Course Outline"),
                         "action_external": False,
                     },
                 ],
             },
             {
-                "short_description": _("Explore edX's Support Tools"),
+                "short_description": _("Other Instructional Material"),
                 "items": [
                     {
-                        "short_description": _("Explore the Studio Help Forum"),
+                        "short_description": _("Verify third-party material is freely available"),
                         "long_description": _(
-                            "Access the Studio Help forum from the menu that appears when you click your user name "
-                            "in the top right corner of Studio."
+                            "Ensure all required third party content is freely available to students (e.g. open source, public"
+                            "domain, cleared/paid by institution, etc.) and integrated into courseware (not linked out)"
```

Comment:
@explorerleslie we just added an item to the MDC that says "Any images used in the course are of consistent and adequate quality", in case you want to add that somewhere around here